deepspeed/runtime/zero/stage2.py

```diff
@@ -1574,14 +1588,14 @@ def backward(self, loss, retain_graph=False):
 
         if self.contiguous_gradients:
             self.ipg_buffer = []
-            buf_0 = torch.empty(int(self.reduce_bucket_size * 4.5),
+            buf_0 = torch.empty(int(self.reduce_bucket_size * 2.5),
                                 dtype=torch.half,
                                 device=torch.cuda.current_device())
             self.ipg_buffer.append(buf_0)
 
             # Use double buffers to avoid data access conflict when overlap_comm is enabled.
             if self.overlap_comm:
-                buf_1 = torch.empty(int(self.reduce_bucket_size * 4.5),
+                buf_1 = torch.empty(int(self.reduce_bucket_size * 2.5),
```

Comment:
I think we ran this with 4.5 to get the best performance.

Reply:
Yes, I reverted this to 4.5, Also, it crashes on larger models with 2.5